<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIBSpectrum</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <widget class="QTabWidget" name="analyzer_tabWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>170</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="analyzer_tabWidget_signal">
       <attribute name="title">
        <string>Signal</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QGroupBox" name="signal_load_groupBox">
          <property name="title">
           <string>Signals</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="signal_load_path_lineEdit">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="signal_choose_dir_pushButton">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="signal_load_path_pushButton">
               <property name="text">
                <string>Load signal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="signal_delete_pushButton">
               <property name="text">
                <string>Dlete signal</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="signal_general_groupBox">
          <property name="title">
           <string>General</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="signal_general_name_label">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="signal_general_name_lineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="signal_general_system_label">
             <property name="text">
              <string>System</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="signal_general_system_comboBox">
             <item>
              <property name="text">
               <string>Bearing</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Shaft</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="signal_sampling_groupBox">
          <property name="title">
           <string>Sampling</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="signal_sampling_frequency_label">
             <property name="text">
              <string>Frequency</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="signal_sampling_frequency_lineEdit"/>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="signal_sampling_frequency_comboBox">
             <item>
              <property name="text">
               <string>Hz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>kHz</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="signal_sampling_period_label">
             <property name="text">
              <string>Period</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="signal_sampling_period_lineEdit"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="signal_sampling_period_comboBox">
             <item>
              <property name="text">
               <string>s</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ms</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>179</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Waveform</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Panner</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="panner_start_label">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="panner_start_label_2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>End</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="panner_end_doubleSpinBox">
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="panner_start_doubleSpinBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="panner_reset_pushButton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Samples</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Time</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>595</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analyzer_tabWidget_spectrum">
       <attribute name="title">
        <string>Spectrum</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>FFT Settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="spectrum_fft_symmetrical_checkBox">
             <property name="text">
              <string>Symmetrical spectrum</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="spectrum_fft_plot_pushButton">
             <property name="text">
              <string>Plot FFT</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>530</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analyzer_tabWidget_tf">
       <attribute name="title">
        <string>Time-Frequency</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="tf_wavelet_settings_groupBox">
          <property name="title">
           <string>Wavelet analysis settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="tf_wavelet_settings_wavelet_label">
             <property name="text">
              <string>Wavelet</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="tf_wavelet_settings_wavelet_comboBox">
             <item>
              <property name="text">
               <string>Morlet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mexican Hat</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Shannon</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Frequency B-Spline</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian 8</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="tf_wavelet_settings_scales_label">
             <property name="text">
              <string>Scales</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="tf_wavelet_settings_scales_spinBox">
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="value">
              <number>64</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="tf_wavelet_plotting_groupBox">
          <property name="title">
           <string>Plotting </string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="tf_wavelet_plotting_palette_label">
             <property name="text">
              <string>Palette</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="tf_wavelet_plotting_palette_comboBox">
             <item>
              <property name="text">
               <string>viridis</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>inferno</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>jet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>plasma</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>magma</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="tf_wavelet_plotting_plot_pushButton">
             <property name="text">
              <string>Plot scaleogram</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="tf_wavelet_plotting_levels_label">
             <property name="text">
              <string>Levels</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="tf_wavelet_plotting_levels_spinBox">
             <property name="value">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Save plot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>490</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="analyzer_tabWidget_mra">
       <attribute name="title">
        <string>Multiresolution Analysis</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Decomposition settings</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Wavelet</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>Morlet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mexican Hat</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Decomposition level</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_2"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Decomposition</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="text">
              <string>Decompose</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>442</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="signals_tableView">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="PlotWidget" name="waveform_widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlotWidget" name="plot_widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_data_file"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuToolbox">
    <property name="title">
     <string>Toolbox</string>
    </property>
    <addaction name="actionBearings"/>
    <addaction name="actionGears"/>
    <addaction name="actionShafts"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuToolbox"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_data_file">
   <property name="text">
    <string>Load data file</string>
   </property>
  </action>
  <action name="actionBearings">
   <property name="text">
    <string>Bearings</string>
   </property>
  </action>
  <action name="actionGears">
   <property name="text">
    <string>Gears</string>
   </property>
  </action>
  <action name="actionShafts">
   <property name="text">
    <string>Shafts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
